src/components/Kv/KvSocialShareButton.vue

```diff
@@ -284,6 +265,27 @@ export default {
 					};
 				}, 500);
 			}
+		},
+		/** displays the share pop up window for whatever service we are sharing on.  */
+		showSharePopUp(destinationUrl, thanksText) {
```

Comment:
This code is mostly from the twitter docs. It just opens a new pop up window with the share interface